src/new-client/src/app/site/[site]/view-availability/page.tsx

```diff
@@ -10,25 +10,33 @@ import { FetchAvailabilityRequest } from '@types';
 import {
   getDetailedMonthView,
   getWeeksInMonth,
-  monthEnd,
-  monthStart,
 } from '@services/viewAvailabilityService';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+
+dayjs.extend(customParseFormat);
```

Comment:
There's already a method for this in the `timeService` file, so you don't need to extend with the plugin in multiple files: 

`export const parseDateString = (dateString: string, format = 'YYYY-MM-DD') => {
  return dayjs.utc(dateString, format, true);
};`

Ideally, I'd have liked all our usages of a third party library to be behind an abstraction so that if we change to a different library at some point it's only one file. Realistically though, that'd just mean creating hundreds of wrapper methods so we have ended up importing dayJs all over the place. 

Because we need the plugin extension for this one I thought it was a good candidate for being given an abstracted wrapper and avoid importing this plugin in multiple files.

Reply:
Turns out I had used this previously then removed it but didn't remove the import.